services/bower/bower-version.service.js

```diff
@@ -38,6 +36,14 @@ class BowerVersion extends BaseBowerService {
       throw new InvalidResponse({ prettyMessage: 'no releases' })
     }
 
+    return version
+  }
+
+  async handle({ packageName }, queryParams) {
+    const data = await this.fetch({ packageName })
+    const includePrereleases = queryParams.include_prereleases !== undefined
+    const version = this.constructor.transform(data, includePrereleases)
+
     return renderVersionBadge({ version })
```

Comment:
transformation logic separated out here to make it easier to use standard unit tests, because we were using mocked service tests to validate this transformation logic before.